Dockerfile

```diff
@@ -18,6 +20,8 @@ COPY . /usr/src/app
 # hack around self-signed ssl cert issue
 RUN export NODE_TLS_REJECT_UNAUTHORIZED=0
 
+RUN npm run build
+
 EXPOSE 5678
 
-CMD [ "npm", "start" ]
+CMD [ "npm", "run", "start-server" ]
```

Comment:
I just added one more script that is run in production. Basically its the same npm start but with different port and without watching functionality